package.json

```diff
@@ -7,25 +7,31 @@
   "sideEffects": false,
   "type": "module",
   "bin": {
-    "giget": "./dist/cli.mjs"
+    "giget": "./bin/index.mjs"
   },
   "exports": {
     ".": {
-      "import": "./dist/index.mjs",
-      "types": "./dist/index.d.ts",
-      "require": "./dist/index.cjs"
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
     }
   },
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.mjs",
```

Comment:
We shall remove this field separately, however there is no reason to do so.

Reply:
I re-add it.